polkadot/runtime/kusama/src/lib.rs

```diff
@@ -316,8 +316,8 @@ impl pallet_balances::Config for Runtime {
 	type MaxReserves = MaxReserves;
 	type ReserveIdentifier = [u8; 8];
 	type WeightInfo = weights::pallet_balances::WeightInfo<Runtime>;
-	type FreezeIdentifier = ();
-	type MaxFreezes = ();
+	type FreezeIdentifier = RuntimeFreezeReason;
+	type MaxFreezes = ConstU32<1>;
```

Comment:
The runtime changes need to be made in the fellowship repo.

Reply:
Kusama and Polkadot runtimes have been removed from `polkadot-sdk` now

Reply:
What do you mean, its still there as much as I can see.